app/jobs/get_phone_metadata_job.rb

```diff
@@ -0,0 +1,19 @@
+class GetPhoneMetadataJob < ApplicationJob
+  queue_as :default
+
+  def perform(intake)
+    return if intake.phone_number.blank?
+
+    metadata = TwilioService.get_metadata(phone_number: intake.phone_number)
+
+    return if metadata.blank?
+
+    if metadata["carrier_name"]
```

Comment:
An idea I want to float - does it make sense to have this be a separate model/table, like `PhoneMetadata`? I personally prefer that since it's not data the client directly is providing. I'm creeping on your PR and have no real context from the work you've done on it, so feel free to ignore as needed.

Reply:
i feel silly making a new table for just two columns? also i think there are a couple of things on intake the client doesn't provide directly so it wouldn't be without precedent